### PR TITLE
Api_Testing part2

### PR DESCRIPTION
**Pull request by NadzeyaKiz**
API testing task part 2 is completed.

### CODE REVIEW

File: code/TestAutomation.Core/Elements/BasePage.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+﻿using OpenQA.Selenium;
+using TestAutomation.Core.Browser;
+
+namespace TestAutomation.Core.Elements
+{
+    public abstract class BasePage
+    {
+        public IWebDriver Driver { get; protected set; }
```
Comment (dnnyhlwdplc) [inline code/TestAutomation.Core/Elements/BasePage.cs:8]: why web part is in PR?

File: code/TestAutomation.Epam.API.Tests/TechTests.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                new TechController(new CustomRestClient()).AddTechItem<TechItemCreatedResponseModel>(techItem)
+                    .TechModel;
+
+            var receivedItem = new TechController(new CustomRestClient()).GetSingleTechItem<TechItemSingleResponseModel>(createdItem.id).TechModel;
+            var receivedItemCapacity = receivedItem.data.Capacity;
+            var receivedItemId = receivedItem.id;
+            Assert.That(techItem.data.Capacity, Is.EqualTo(receivedItemCapacity), 
+                $"The capacity of the created object {receivedItemCapacity} is equal to initial one {techItem.data.Capacity}");
+
+            new TechController(new CustomRestClient()).DeleteSingleCreatedItem(receivedItemId);
```
Comment (dnnyhlwdplc) [inline code/TestAutomation.Epam.API.Tests/TechTests.cs:63]: please verify that item is deleted

File: code/TestAutomation.Epam.API/Controllers/TechController.cs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        /// </summary>
+        /// <typeparam name="T"><see cref="TechItemSingleResponseModel"/></typeparam>        
+        /// <returns>response typeof <see cref="RestResponse"/></returns>
+        public (RestResponse Response, T? TechModel) UpdateTechItem<T>(object model, string techId)
+        {
+            return Patch<T, TechItemRequestModel>(string.Format(SingleTechResource, techId), model);
+        }
+    }
+}
+
```
Comment (dnnyhlwdplc) [inline code/TestAutomation.Epam.API/Controllers/TechController.cs:68]: please remove extra spaces :)

**Review by dnnyhlwdplc — APPROVED** [review]

File: code/TestAutomation.Epam.API/Controllers/TechController.cs (hunk 3 above)
Comment (NadzeyaKiz) [reply code/TestAutomation.Epam.API/Controllers/TechController.cs:68]: done

File: code/TestAutomation.Epam.API.Tests/TechTests.cs (hunk 2 above)
Comment (NadzeyaKiz) [reply code/TestAutomation.Epam.API.Tests/TechTests.cs:63]: done

File: code/TestAutomation.Core/Elements/BasePage.cs (hunk 1 above)
Comment (NadzeyaKiz) [reply code/TestAutomation.Core/Elements/BasePage.cs:8]: by mistake commented it here

**Pull request merged**